src/components/EditBeach/index.js

```diff
@@ -0,0 +1,230 @@
+/* eslint-disable @next/next/no-img-element */
+
+import styles from "components/EditBeach/EditBeach.module.css"
+import { useRouter } from "next/router";
+import React from "react";
+import { rootDomain } from "lib/constants";
+import SignupInput from "components/SignupInput";
+import { useDropzone } from "react-dropzone";
+import { v4 as uuidv4 } from 'uuid';
+import PrimaryButton from "components/PrimaryButton";
+import { toaster } from 'evergreen-ui';
+import Cookies from 'js-cookie';
+const BeachEditComponent = () => {
+    const [oldData, setOldData] = React.useState('');
```

Comment:
i wonder if it would be better to make just an oldData and newData state object, and then manage all the input fields via the key name. i think this might make it a bit simpler and we won't need as much if statements at the end. we can go over what i mean in a call if that's confusing